app/src/main/res/values/strings.xml

```diff
@@ -1654,10 +1655,32 @@
     <string name="patroller_diff_feedback_dialog_option_neutral">Neutral</string>
     <string name="patroller_diff_feedback_dialog_option_unsatisfied">Unsatisfied</string>
     <string name="patroller_diff_feedback_dialog_feedback_title">How can we improve the feature?</string>
+    <string name="patroller_diff_menu_edit_history_title">Article edit history</string>
     <string name="patroller_diff_feedback_dialog_input_hint">Feedback</string>
     <string name="patroller_diff_feedback_dialog_submit">Submit</string>
     <string name="patroller_diff_feedback_submitted_snackbar">Feedback submitted.</string>
     <string name="patroller_diff_feedback_tooltip">Tap on overflow menu to submit feedback anytime under \'Problem with feature\'.</string>
+    <string name="patroller_saved_message_title_1">Vandalism warning</string>
```

Comment:
Can we make the names of the string resources more self-documenting? Ideally we should avoid ordinal numbers to distinguish them, and instead use meaningful names, such as `patroller_saved_message_title_npov`, `patroller_saved_message_title_copyvio`, and so on.